themes/ddbasic/sass/module/_carousel.scss

```diff
@@ -461,3 +453,27 @@ $carousel-tab-color:                       $white;
     font-size: 80%;
   }
 }
+
+.content-wrapper {
```

Comment:
We try to override this style: https://github.com/easySuite/ding2/blob/development/themes/ddbasic/css/ddbasic.styles.css#L3468.

The only other solution is to add _!important_ rule. I don't think that it is ok.